common/models/QoL.ts

```diff
@@ -35,11 +35,19 @@ export type QolSurveyResults = {
     [dom: string]: number,
 };
 
+export enum QolType {
+    Onboarding = 'ONBOARDING',
+    Monthly = 'MONTHLY',
+}
+
 export type PartialQol = {
     questionNum: number,
     domainNum: number,
     scores: QolSurveyResults,
     isFirstTimeQol: boolean,
+    startDate: number,
+    lastResponseDate: number,
+    questionCompletionDates: number[],
```

Comment:
is `lastResponseDate` redundant if we can infer it by taking the tail element of `questionCompletionDates`?

Reply:
yes good point